src/SFA.DAS.Commitments.Api.UnitTests/Orchestrators/Mappers/WhenMappingAnApprenticeshipResult .cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using AutoFixture;
+using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.Commitments.Api.Orchestrators.Mappers;
+using SFA.DAS.Commitments.Application.Rules;
+using SFA.DAS.Commitments.Domain;
+using SFA.DAS.Commitments.Domain.Entities;
+using OrganisationType = SFA.DAS.Common.Domain.Types.OrganisationType;
+
+namespace SFA.DAS.Commitments.Api.UnitTests.Orchestrators.Mappers
+{
+    [TestFixture]
+    public class WhenMappingAnApprenticeshipResult
+    {
+        private ApprenticeshipMapper _mapper;
+        private List<ApprenticeshipResult> _apprenticeships;
+
+        [SetUp]
+        public void Setup()
+        {
+            var fixture = new Fixture();
+            _apprenticeships = fixture.Create<List<ApprenticeshipResult>>();
+            _mapper = new ApprenticeshipMapper();
+        }
+
+        [Test]
+        public void ThenMappingListCompletesSuccessfully()
+        {
+            Assert.DoesNotThrow(() => _mapper.MapFrom(_apprenticeships));
+        }
+
+        [Test]
+        public void ThenMapsApprenticeshipValuesCorrectly()
+        {
+            var result = _mapper.MapFrom(_apprenticeships).ToList();
+
+            Assert.AreEqual(result.Count, _apprenticeships.Count);
```

Comment:
The expected param comes first. It will give a confusing message is the assert fails!